docs/dev/annual_updates.rst

```diff
@@ -106,44 +107,29 @@ pages, or columns) then create new mappings to track that information over time.
     In all of the the above CSV files we use a value of ``-1`` to indicate that the data
     does not exist in a given year.
 
-B. FERC Form 1
```

Comment:
This got bumped to section 4: `Update Table & Column Transformations` because it is more closely linked to the transform process. But if you think it belongs in Section 2: `Map the Structure of the New Data` lmk! That is where the info about EIA skip_rows etc. is. The section for FERC 714 does reference `pudl.extract.ferc714.TABLE_FNAME` which is pretty similar to `pudl.extract.ferc1.TABLE_NAME_MAP_FERC1 `...

Reply:
I personally think it is more map-y, but its a fuzzy line and I think it is fine where it is! If the EIA mapping is mostly an extract-stage mapping then maybe that is a good distinction/line to draw.